doc/flame/examples/lib/opacity_effect.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:doc_flame_examples/ember.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/experimental.dart';
+import 'package:flame/game.dart';
+
+class OpacityGame extends FlameGame with HasTappableComponents {
+  bool reset = false;
+  @override
+  Future<void> onLoad() async {
+    final ember = EmberPlayer(
+      position: canvasSize / 3,
```

Comment:
I wasn't able to get the sprite fully into the center. This was the closest generalized value I could find. Lemme know if this is fine

Reply:
Set the anchor to `Anchor.center` and set the position to (game) size/2